src/Stripe.net/Services/Account/StripeAccountCreateOptions.cs

```diff
@@ -7,7 +7,7 @@ public class StripeAccountCreateOptions : StripeAccountSharedOptions
         [JsonProperty("country")]
         public string Country { get; set; }
 
-        [JsonProperty("managed")]
-        public bool? Managed { get; set; }
+        [JsonProperty("type")]
+        public string Type { get; set; }
```

Comment:
@anelder-stripe I thought the same thing when we named the property internally but Type is used elsewhere for example on [Balance Transaction](https://github.com/jaymedavis/stripe.net/blob/master/src/Stripe.net/Entities/StripeBalanceTransaction.cs#L40) so I think we're good on this one.